osf/models/mixins.py

```diff
@@ -2021,9 +2021,8 @@ def _get_spam_content(self, saved_fields):
         :param saved_fields: set
         :return: str
         """
-        spam_fields = self.get_spam_fields(saved_fields)
```

Comment:
There are some fields that shouldn't be spam-checked, and it makes more sense to get only spam fields here than, say, at every call to `check_spam`.

Reply:
yeah I meant to revert this, sorry!